app/config/locale/translations/fr.json

```diff
@@ -3,30 +3,30 @@
     "settings.locale": "fr",
     "settings.direction": "ltr",
     "emails.sender": "Équipe %s",
-    "emails.verification.subject": "",
-    "emails.verification.hello": "",
-    "emails.verification.body": "",
-    "emails.verification.footer": "",
-    "emails.verification.thanks": "",
-    "emails.verification.signature": "",
-    "emails.magicSession.subject": "",
-    "emails.magicSession.hello": "",
-    "emails.magicSession.body": "",
-    "emails.magicSession.footer": "",
-    "emails.magicSession.thanks": "",
-    "emails.magicSession.signature": "",
-    "emails.recovery.subject": "",
-    "emails.recovery.hello": "",
-    "emails.recovery.body": "",
-    "emails.recovery.footer": "",
-    "emails.recovery.thanks": "",
-    "emails.recovery.signature": "",
-    "emails.invitation.subject": "",
-    "emails.invitation.hello": "",
-    "emails.invitation.body": "",
-    "emails.invitation.footer": "",
-    "emails.invitation.thanks": "",
-    "emails.invitation.signature": "",
+    "emails.verification.subject": "Vérification du compte",
+    "emails.verification.hello": "Bonjour {{name}}",
+    "emails.verification.body": "Suivez ce lien pour vérifier votre adresse mail.",
+    "emails.verification.footer": "Si vous n'avez pas demandé à vérifier cette adresse mail, vous pouvez ignorer ce message.",
+    "emails.verification.thanks": "Merci",
+    "emails.verification.signature": "Équipe {{project}}",
+    "emails.magicSession.subject": "Connexion",
+    "emails.magicSession.hello": "Bonjour",
```

Comment:
Please use the same punctuation as seen in the English translation https://github.com/appwrite/appwrite/blob/master/app/config/locale/translations/en.json#L13